core/src/main/java/bisq/core/provider/fee/FeeService.java

```diff
@@ -157,14 +158,15 @@ public void onSuccess(@Nullable Tuple2<Map<String, Long>, Map<String, Long>> res
                         epochInSecondAtLastRequest = timeStampMap.get("bitcoinFeesTs");
                         final Map<String, Long> map = result.second;
                         txFeePerVbyte = map.get("BTC");
+                        minFeePerVByte = map.get("btcMinTxFee");
```

Comment:
Just because this map key `btcMinTxFee` is used in at least one other class, should it be defined as a static field in one class to avoid typos?   I'm not sure, but it's something to consider...

Reply:
Yes great idea, in fact I did run into a scenario like that during testing.  So I will extract those hardcoded strings shared between Bisq and PriceNode into `Common/Config.java`.  Thanks.